pkg/doc/verifiable/example_credential.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import "encoding/json"
+
+//UniversityDegree university degree
+type UniversityDegree struct {
+	Type       string
+	Name       string
+	College    string
+	University string
+}
+
+//UniversityDegreeSubject subject of university degree
+type UniversityDegreeSubject struct {
+	ID     string
+	Name   string
+	Spouse string
+
+	Degree UniversityDegree
+}
+
+//UniversityDegreeCredential University Degree credential, from examples of https://w3c.github.io/vc-data-model
+type UniversityDegreeCredential struct {
+	c Credential
+
+	Subject *UniversityDegreeSubject
+}
+
+type rawDegree struct {
```

Comment:
I think we would benefit from removing these `raw*` structs altogether from this test and have the non-raw structs hold the json tags. The reason is that I wouldn't want to give the wrong impression and have users think they necessarily have to follow this roundabout way of unmarshalling the credential when a simpler option is available.

Reply:
OK!